components/admin_console/ldap_settings.jsx

```diff
@@ -83,6 +84,44 @@ export default class LdapSettings extends AdminSettings {
         );
     }
 
+    renderJobData(job) {
+        var matterMostUsers = '0';
```

Comment:
Should probably change "matterMostUsers" to "mattermostUsers" otherwise IT might go crazy :P

Reply:
will fix.